inc/CLog.make

```diff
@@ -26,10 +26,9 @@ function(CLOG_GENERATE_TARGET)
 
         add_custom_command(
             OUTPUT ${ARG_CLOG_FILE} ${ARG_CLOG_C_FILE}
-            DEPENDS CLOG
             DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${arg}
-            COMMENT "CLOG: ${CMAKE_CLOG_BINS_DIRECTORY}/clog --readOnly -p ${CMAKE_CLOG_CONFIG_PROFILE} --scopePrefix ${library} -c ${CMAKE_CLOG_CONFIG_FILE} -s ${CMAKE_CLOG_SIDECAR_DIRECTORY}/clog.sidecar -i ${CMAKE_CURRENT_SOURCE_DIR}/${arg} -o ${ARG_CLOG_FILE}"
-            COMMAND ${CMAKE_CLOG_BINS_DIRECTORY}/clog --readOnly -p ${CMAKE_CLOG_CONFIG_PROFILE} --scopePrefix ${library} -c ${CMAKE_CLOG_CONFIG_FILE} -s ${CMAKE_CLOG_SIDECAR_DIRECTORY}/clog.sidecar -i ${CMAKE_CURRENT_SOURCE_DIR}/${arg} -o ${ARG_CLOG_FILE}

```

Comment:
Would this break msquic if we continue to consume this file directly from clog?

Reply:
No. This change we're doing in msquic as well. This is how this is consumed directly as a global tool.

Also, we wouldn't be pulling in the submodule for just this single file. This would be a file we would keep in our repo directly, matching the version of clog we use.

Reply:
Ok. So the design then, if folks want to build clog as part of their pipeline, is to build clog, then add it to their path and then run their own build that uses clog?

Reply:
Yeah. Although I might need to think about that more, because I don't think cmake will be able to detect an update to the binary, so it would not regenerate the headers upon a clog update. Hmm.

Reply:
I think required a forced rebuild if you update clog isn't the end of the world.